content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -300,11 +300,12 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 ### Volume Mode
 
 {{< feature-state for_k8s_version="v1.9" state="alpha" >}}
```

Comment:
question for docs: do we still need the alpha info?

Reply:
This should be updated because the pr is targeting dev-1.13 branch.